alphadia/workflow/peptidecentric.py

```diff
@@ -366,18 +366,33 @@ def get_optimization_lock(self):
                 break
 
         optimization_lock_eg_idxes = self.elution_group_order[:final_stop_index]
-
         optimization_lock_library_precursor_df = self.spectral_library._precursor_df[
             self.spectral_library._precursor_df["elution_group_idx"].isin(
                 optimization_lock_eg_idxes
             )
         ]
 
+        randomized_idx = np.random.permutation(
```

Comment:
in order to increase reproducibility: should we fix the seed? or at least enable fixing it (e.g. via config)?